modules/tests/test/copy_template_test.go

```diff
@@ -141,28 +99,49 @@ var _ = Describe("Copy template task", func() {
 				TaskRunTestConfig: testconfigs.TaskRunTestConfig{
 					ServiceAccount: CopyTemplateServiceAccountName,
 				},
-				TaskData: testconfigs.CopyTemplateTaskData{
-					Template:                cirrosTemplate,
-					SourceTemplateName:      cirrosTemplate.Name,
-					SourceTemplateNamespace: cirrosTemplate.Namespace,
-					TargetTemplateName:      newTemplateName,
-					TargetTemplateNamespace: f.TestNamespace,
-				},
-			}, cirrosTemplate.Namespace,
-			),
+				TaskData: testconfigs.CopyTemplateTaskData{},
+			}),
 			table.Entry("should create template in different namespace", &testconfigs.CopyTemplateTestConfig{
 				TaskRunTestConfig: testconfigs.TaskRunTestConfig{
 					ServiceAccount: CopyTemplateServiceAccountName,
 					LimitTestScope: ClusterTestScope,
 				},
 				TaskData: testconfigs.CopyTemplateTaskData{
-					Template:                cirrosTemplate,
-					SourceTemplateName:      cirrosTemplate.Name,
-					SourceTemplateNamespace: cirrosTemplate.Namespace,
-					TargetTemplateName:      newTemplateName,
 					TargetTemplateNamespace: openshiftNamespace,
```

Comment:
We should not polute other namespaces with test resources. There is a deploy DeployTargetNS a TestTargetNS for this use case.